ontology/uco-gufo.ttl

```diff
@@ -48,6 +58,21 @@ drafting:Quality
 	rdfs:seeAlso <https://github.com/ucoProject/UCO/issues/535> ;
 	.
 
+drafting:Vehicle
+	a owl:Class ;
```

Comment:
Assuming we're talking about vehicles as concrete things (as opposed to abstract things and relations), Vehicle is of type `Kind` (or `Subkind`) if one considers it a rigid sortal; or of type `Category` or `Mixin` otherwise, since it definitely is an Endurant. 

On categorising (typing) the Vehicle in the. context of UCO, which of those types does it fit best?

Reply:
For the sake of UCO Issue 567, Vehicle means the more-concrete things: car, truck, boat, bike.

Reply:
I'd prefer to leave the gufo-typing of `Vehicle` out of this PR.

Reply:
You can prefer that, but the reasoner will decide otherwise 😃 . But since UCO nor CASE do reasoning, your preference is safe 👍